generator/src/main/java/com/scottlogic/deg/generator/generation/GenerationConfig.java

```diff
@@ -76,7 +77,7 @@ public String toString() {
     }
 
     public enum TreeWalkerType {
-        CARTESIAN_PRODUCT(Constants.WalkerTypes.CARTESIAN_PRODUCT),
+        CARTESIAN_PRODUCT(GenerateCommandLine.defaultTreeWalkerType),
```

Comment:
This should stay as `Constants.WalkerTypes.CARTESIAN_PRODUCT`, and `GenerateCommandLine` should reference `Constants.WalkerTypes.CARTESIAN_PRODUCT` instead of the other way around